routes/resourcePages.js

```diff
@@ -140,10 +140,10 @@ async function renameResourcePage (req, res, next) {
   res.status(200).json({ resourceName, pageName: newPageName, content, sha: newSha })
 }
 router.get('/:siteName/resources/:resourceName', attachRouteHandlerWrapper(listResourcePages))
-router.post('/:siteName/resources/:resourceName/pages', attachRouteHandlerWrapper(createNewResourcePage))
+router.post('/:siteName/resources/:resourceName/pages', attachRollbackRouteHandlerWrapper(createNewResourcePage))
 router.get('/:siteName/resources/:resourceName/pages/:pageName', attachRouteHandlerWrapper(readResourcePage))
 router.post('/:siteName/resources/:resourceName/pages/:pageName', attachRouteHandlerWrapper(updateResourcePage))
-router.delete('/:siteName/resources/:resourceName/pages/:pageName', attachRouteHandlerWrapper(deleteResourcePage))
+router.delete('/:siteName/resources/:resourceName/pages/:pageName', attachRollbackRouteHandlerWrapper(deleteResourcePage))
 router.post('/:siteName/resources/:resourceName/pages/:pageName/rename/:newPageName', attachRouteHandlerWrapper(renameResourcePage))
```

Comment:
`renameResourcePage` should use `attachRollbackRouteHandlerWrapper` as well in case the creation of the new file succeeds but the delete of the old file fails

Reply:
Good catch, resolved in https://github.com/isomerpages/isomercms-backend/pull/91/commits/967eaa01b0087bbd047e511dc28ccc8993a1eadf